integration/src/main/java/scripts/utils/TestUtils.java

```diff
@@ -0,0 +1,11 @@
+package scripts.utils;
+
+import java.util.Random;
+
+public class TestUtils {
+  private static final Random RANDOM = new Random();
+
+  public static String appendRandomNumber(String string) {
```

Comment:
since there's only one usage of it, i think we should keep this in the caller's class for now.

Reply:
I'd prefer to leave it here. If it exists, people are more likely to use it, rather than manually write this code themselves ([example](https://github.com/DataBiosphere/terra-cli/blob/5780f4bc06115e4768b2e9296882d9022b756485/src/test/java/unit/CloneWorkspace.java#L183)).